### PR TITLE
Minor editorial changes

### PR DESCRIPTION
**Pull request by gorryfair**
Proposed Minor editorial work

### CODE REVIEW

File: draft-ietf-ccwg-rfc5033bis.md [hunk 1]
```diff
@@ -182,10 +182,10 @@ appropriate congestion control scheme.
 Rather, the document provides
 a set of criteria that should be considered and weighed by the
 developers of congestion control algorithms and by the IETF
-in the context of each proposal.
-The high-order criteria for any new
-proposal is that a serious scientific study of the pros and cons of
-the proposal needs to have been done before a proposal is
+in the context of each .
+The high-order criteria for any new alternate congestion control
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:186]:
```suggestion
The high-order criteria for any new congestion control
```

File: draft-ietf-ccwg-rfc5033bis.md [hunk 2]
```diff
@@ -182,10 +182,10 @@ appropriate congestion control scheme.
...
+is that a serious scientific study of the pros and cons of
+the  needs to have been done before a proposal is
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:188]:
```suggestion
is that a serious scientific study of the pros and cons needs to have been done before a proposal is
```

File: draft-ietf-ccwg-rfc5033bis.md [hunk 3]
```diff
@@ -464,7 +464,9 @@ discussed in Sections 10.3 and 10.4 of {{?RFC4782}} (Quick-Start).
 # General Use {#general-use}
 
 The criteria in {{evaluation-criteria}} will be evaluated in the
-following scenarios. Unless a proposal is explicitly forbidden on the
+following scenarios.
+Unless an alternate congestion control
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:468]:
```suggestion
Unless a congestion control
```

File: draft-ietf-ccwg-rfc5033bis.md [hunk 4]
```diff
@@ -558,6 +561,9 @@ delay product, well below the standard operating range of most Internet flows.
 
 Satellite links often have delays longer than typical for wired paths
 {{?RFC2488}} and high delay bandwidth products {{?RFC3649}}.
+Many systems
+use dynamic capacity assignment that can result in variation of the delay
+and the capacity over timescales of the order of the path RTT.
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:566]: Honest question: Is delay variation due to "dynamic capacity assignment" meaningfully different from queueing delay.

**Review by martinduke — CHANGES_REQUESTED** [review]

File: draft-ietf-ccwg-rfc5033bis.md (hunk 4 above)
Comment (huitema) [reply draft-ietf-ccwg-rfc5033bis.md:565]: That's a very old debate. On one hand, in CSMA systems, the mean time to wait before transmission is indeed linked to the network load. Increased delays are a sign of congestion. But that's not always the case. In mobile networks, a station may well move to a different cell. The delay may be longer, if the new base station is further away, or needs to poll more clients. The data rate may or may not be lower, depending of frequencies and bandwidth available. The old logic breaks.

Comment (gorryfair) [reply draft-ietf-ccwg-rfc5033bis.md:565]: The case I was thinking of was Bandwidth on Demand rather than CSMA, where capacity requests are made to an allocator and capacity is assigned from a resource pool. The two main complications are: (1) The time to respond is not related to the traffic load: it is a function of the link RTT, and the allocation interval (often related to the super frame size in the systems I know of), but also dependent on the design of the method ... these have nothing to do with the queue occupancy (2) The available capacity from the resource pool depends on what can be used by the specific system ... which may be a subset of the total resource ... and also the load on that resource by other systems, propagation conditions, implemented waveforms, etc.

In short: when there is a queue, I see dynamic capacity assignment can cause delay, but the length of the delay is a function of an external control system, not the level of congestion for the flow - i.e. the delay can be transient and have no bearing on the availability of capacity next RTT. Whereas, for a FIFO queue, the queueing delay can have a strong correlation with the availability of capacity next RTT.

File: draft-ietf-ccwg-rfc5033bis.md [hunk 5]
```diff
@@ -182,10 +182,9 @@ appropriate congestion control scheme.
 Rather, the document provides
 a set of criteria that should be considered and weighed by the
 developers of congestion control algorithms and by the IETF
-in the context of each proposal.
-The high-order criteria for any new
-proposal is that a serious scientific study of the pros and cons of
-the proposal needs to have been done before a proposal is
+in the context of each .
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:185]: Remove space, or restore the word 'proposal'?

**Review by martinduke — APPROVED** [review]

File: draft-ietf-ccwg-rfc5033bis.md (hunk 4 above)
Comment (gorryfair) [reply draft-ietf-ccwg-rfc5033bis.md:565]: So... let's see of what actually ought to be written for *evaluation* ?
